packages/workbox-build/src/lib/generate-sw.js

```diff
@@ -18,6 +18,8 @@ const constants = require('./constants');
  * `globPatterns` against.
  * @param {Array<String>} input.globPatterns Files matching against any of
  * these glob patterns will be included in the file manifest.
+ *
+ * Defaults to ['**\/*.{js,css}']
```

Comment:
no you can't because JSDoc get's upset. Will look into filter this out in the code snippets somehow.